src/job_cache/job_cache.cpp

```diff
@@ -105,10 +107,6 @@ static bool daemonize(std::string dir) {
     replace_fd(STDERR_FILENO, err_log_fd->get());
   }
 
-  wcl::log::clear_subscribers();
-  wcl::log::subscribe(std::make_unique<wcl::log::FormatSubscriber>(std::cout.rdbuf()));
-  wcl::log::info("Reinitialized logging for job cache daemon")();
-
```

Comment:
Are we sure we're not changing what gets logged where with this?

Reply:
Ok I thought through it and yeah it does mean that many error logs might show up in wake.log instead of in the shared cache but I think that's ok since this is launching code.